code-conventions.md

```diff
@@ -6,41 +6,80 @@ servers; security matters. Following these guidelines is *vital* for all new
 code. Contributors should also see the dedicated
 [security document](security.md).
 
-* **NO MALICIOUS CODE** - I think you already knew this
-  ```java
-  Runtime.getRuntime().exec("rm -rf /"); // Really BAD
-  ```
-* **DO NOT** submit secrets (passwords, etc.) to the Git
-  ```java
-  String myApiToken = "f6dd0c930676da532136fbef805ef1553ebaca83"; // Really BAD
-  // Though this is not a real token - can you find what it is?
-  ```
-  - If this happens and somehow gets past code review, you'll have to change
-    your secrets; we are generally unwilling to 'force push' anything
-* Client or server crashes/freezes shouldn't be easy to create from scripts
-  ```java
-  if (matchedPattern == 0)
-      wait(1000); // BAD
-  ```
-  - This is not to say that you can't currently crash the client, but avoid
-    adding any more ways to cause trouble
-  - Be especially wary of things people might accidentally use
-* Obviously bad ideas (such as MD5 hashing) should not look safe to scripters
-  ```java
-  MessageDigest md5 = MessageDigest.getInstance("MD5"); // BAD
-  // ... although this is a snippet from Skript source code. Legacy stuff!
-  ```
-  - Make it clear when scripts are doing something insecure or dangerous
-  - Exception: consider twice before breaking API exposed to scripts
-* File or network IO, for now, should not be accessible from scripts
-  ```java
-  String fileName = file.getSingle(e); // Bad
-  ```
-* Code contributed must be licensed under GPLv3, by *you*
-  - Third party code under compatible licenses may be accepted in some cases
-  - We'll forward any complaints about code licensing to people who added it
+Our users have an expectation of security and trust; it is important we make sure our code is safe.
+
+
+### Malicious Code
+No contributions should contain malicious code designed to harm the user or their machine. 
+Nothing should intend to damage or delete system files, the user's personal files or unrelated parts of the Minecraft server.
+```java
+Runtime.getRuntime().exec("rm -rf /"); // bad, don't do this
+```
+While we expect contributors to use common sense, a general rule would be that contributions should not modify or delete
+ resources that do not belong to Skript or could be expected to be modified by Skript (e.g. minecraft world data).
```

Comment:
The example here can be interpreted as 'Skript should be able to place blocks in the world'

Reply:
I'm not sure I understand, can you clarify?

Reply:
Discussed on Discord, this probably needs to be phrased more clearly.